pkg/admission/reservedcrdgroups/admission.go

```diff
@@ -70,7 +70,7 @@ func (o *reservedCRDGroups) Validate(ctx context.Context, a admission.Attributes
 		return nil
 	}
 
-	if strings.HasSuffix(crd.Spec.Group, "kcp.dev") {
+	if strings.HasSuffix(crd.Spec.Group, "kcp.dev") && crd.Spec.Group != "initialization.tenancy.kcp.dev" {
```

Comment:
We need something durable here, this works for now ...

Reply:
/hold

This is wrong.

Reply:
Requires #1386